lib/main_common.pm

```diff
@@ -693,6 +693,8 @@ sub remove_common_needles {
 
     unregister_needle_tags("ENV-VIDEOMODE-text") unless check_var("VIDEOMODE", "text");
     unregister_needle_tags('ENV-ARCH-s390x')     unless check_var('ARCH',      's390x');
+    # Only for container tests
+    unregister_needle_tags('ENV-UBUNTU') unless get_var('HDD_1') =~ /ubuntu/;
```

Comment:
The needle has:

```suggestion
    unregister_needle_tags('ENV-UBUNTU-1') unless get_var('HDD_1') =~ /ubuntu/;
```

Reply:
Either you are too fast or i am too late. :)